routes/articles.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const articleDB = require('../db/articles.js');
+const middleware = require('../middleware/validate');
+const database = articleDB.getArticles().articles;
```

Comment:
You shouldn't try to use the array inside articleDb like this. You should stick with the methods expose through the module.exports.